scripts/download_cams_pm25.py

```
@@ -0,0 +1,267 @@
+except ImportError:
+    TQDM_AVAILABLE = False
+    print("Note: Install tqdm for progress bars: pip install tqdm")
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+
+async def download_cams_data(
```

Ruff (0.12.2)

`download_cams_data` is too complex (14 > 10)

(C901)